Chatto/Source/ChatController/AccessoryViewRevealer.swift

```diff
@@ -73,9 +74,15 @@ class AccessoryViewRevealer: NSObject, UIGestureRecognizerDelegate {
     }
 
     private func revealAccessoryView(atOffset offset: CGFloat) {
+        // Find max offset (cells can have slighlty different timestamp size ( 3.00 am vs 11.37 pm )
+        let cells: [AccessoryViewRevealable] = self.collectionView.visibleCells().filter({$0 is AccessoryViewRevealable}).map({$0 as! AccessoryViewRevealable})
```

Comment:
It doesn't compile unfortunately: 'AccessoryViewRevealable' is not a subtype of 'UICollectionViewCell'